build/templates/test_converters.py.mako

```diff
@@ -0,0 +1,297 @@
+# This file was generated
```

Comment:
I don't think this needs to remain a .mako file.
Just turn it into a regular .py file and source it in https://github.com/ni/nimi-python/tree/master/src/nifake/unit_tests

Reply:
Hmm, you're right. No need to use mako at all 🤦‍♂️ 

Updated the PR.